internal/builder/builder_test.go

```diff
@@ -384,6 +384,20 @@ func testBuilder(t *testing.T, when spec.G, it spec.S) {
 				)
 			})
 
+			it("creates the build-config dir", func() {
+				h.AssertNil(t, subject.Save(logger, builder.CreatorMetadata{}))
+				h.AssertEq(t, baseImage.IsSaved(), true)
+
+				layerTar, err := baseImage.FindLayerWithPath("/cnb/build-config")
+				h.AssertNil(t, err)
+				h.AssertOnTarEntry(t, layerTar, "/cnb/build-config",
+					h.IsDirectory(),
+					h.HasOwnerAndGroup(0, 0),
```

Comment:
Hmm... I don't think we ever specified who the owner should be. Is it possible to check who owns for example `/cnb/buildpacks`? It should probably be the same for this directory and its children.

Reply:
Update: it looks like `/cnb/buildpacks` is root owned also, so this is correct :)